conf/Makefile.chibios

```diff
@@ -246,9 +251,9 @@ TCSRC =
 TCPPSRC =
 
 # List ASM source files here
-ASMSRC = $(STARTUPASM) $(PORTASM) $(OSALASM)
+ASMXSRC = $(STARTUPASM) $(PORTASM) $(OSALASM)
```

Comment:
yep, F7 support is stable branch is buggy, we found a bug that prevent the RTC to start and then the logging is not working correctly (which we consider as a major need when using ChibiOS).